cms/envs/common.py

```diff
@@ -1065,7 +1065,10 @@
 ################################# CELERY ######################################
 
 # Auto discover tasks fails to detect contentstore tasks
-CELERY_IMPORTS = ('cms.djangoapps.contentstore.tasks')
+CELERY_IMPORTS = (
+    'cms.djangoapps.contentstore.tasks',
+    'openedx.core.djangoapps.bookmarks.tasks'
```

Comment:
Does this run in both CMS and LMS contexts?  Do we need both?

Reply:
It only runs in CMS context and is called from inside the course_published signal
https://github.com/edx/edx-platform/blob/353d006839c21ebde789f12851e5cc9cd425435a/openedx/core/djangoapps/bookmarks/signals.py#L15-L23

I have removed the import from lms settings